src/helpers/eventInstances/daily.ts

```diff
@@ -0,0 +1,50 @@
+import type mongoose from "mongoose";
+import type {
+  InterfaceEvent,
+  InterfaceOrganization,
+  InterfaceUser,
+} from "../../models";
+import { Event } from "../../models";
+import type { MutationCreateEventArgs } from "../../types/generatedGraphQLTypes";
+import { eachDayOfInterval, format } from "date-fns";
+
+interface InterfaceRecurringEvent extends MutationCreateEventArgs {
+  startDate: Date;
+  creatorId: mongoose.Types.ObjectId;
+  admins: mongoose.Types.ObjectId[];
+  organization: mongoose.Types.ObjectId;
+}
+
+export async function generateEvents(
+  args: Partial<MutationCreateEventArgs>,
```

Comment:
Why are using `Partial` here? How would you create a valid event if any of the listed `InterfaceRecurrignEvent` args are not passed?